index.js

```diff
@@ -82,6 +83,7 @@ function Bankai (entry, opts) {
     if (self.optimize) {
       b.transform(unassertify, { global: true })
       b.transform(yoyoify, { global: true })
+      b.transform(envify, { global: true })
```

Comment:
Oh yeah to do that we would have to use `envify/custom`. Anyway, I'm a bit lost right now haha. `envify` replace env vars but we have to set them in some where, so if we are not setting `NODE_ENV` then there is no way to avoid `NODE_ENV=production` or I'm missing something?

Other thing we could do is maybe define a special bankai env var, something like `BANKAI_ENV` and document it properly